examples/DeviceInfo/DeviceInfo.ino

```diff
@@ -3,7 +3,7 @@
 #define debugSerial Serial
 #define loraSerial Serial1
 
-TheThingsNetwork ttn;
```

Comment:
This (and the other places) should be left as `ttn`, not `ttu` as it is currently in e.g. https://github.com/TheThingsNetwork/arduino-library/blob/master/examples/DeviceInfo/DeviceInfo.ino#L6